scalafix-core/shared/src/main/scala/scalafix/internal/rule/ProcedureSyntax.scala

```diff
@@ -12,7 +12,7 @@ case object ProcedureSyntax
     extends Rule(
       RuleName("ProcedureSyntax").withDeprecatedName(
         "ExplicitUnit",
-        "Use ProcedureSyntax instead",
+        "Use explicit : Unit = instead",
```

Comment:
same as above, this `withDeprecatedName` is used to provide alternate deprecated names for rules.